src/commands/language.ts

```diff
@@ -19,7 +19,8 @@ createCommand({
   guildOnly: true,
   permissionLevels: [PermissionLevels.MEMBER],
   execute: async function (message) {
-    const currentLanguageId = getCurrentLanguage(message.guildID) || 'en_US';
+    //@ts-ignore
```

Comment:
Why is this?

Reply:
My Typescript linter does a warning that the `message.guild` could possibly be undefined. This is however not possible since this command is guilldOnly. This comment disables the warning.

Reply:
I don't get this `|| 'en_US';`

Reply:
Then just make this guildID in the function optional.

Reply:
You are right. this is not needed.

Reply:
Actually i fixed this in one of the commits., Not sure why it cannot be seen here.

Reply:
This was fixed already. I do not know why it cannot be seen here. Can i resolve this?

Reply:
No since it is in the file

Reply:
Nevermind, I get what you mean now. Gonna remove the comment asap.